app/models/manageiq/providers/google/cloud_manager.rb

```diff
@@ -95,7 +95,16 @@ def gce
     @gce ||= connect(:service => "GCE")
   end
 
-  # Operations
+  # Unsupported Operations
+  def vm_shelve(vm, _options = {})
```

Comment:
@Ladas https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/vm/operations/power.rb#L2-L8

are those 2 validate methods only for UI?
and does it make sense to return something like this in this case?
```
{:available => false,  :message => "not supported"}
```